common/src/main/java/com/wynntils/models/activities/ActivityTrackerScoreboardPart.java

```diff
@@ -13,10 +13,12 @@
 import com.wynntils.handlers.scoreboard.type.SegmentMatcher;
 import java.util.List;
 import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import net.minecraft.ChatFormatting;
 
 public class ActivityTrackerScoreboardPart extends ScoreboardPart {
     private static final SegmentMatcher TRACKER_MATCHER = SegmentMatcher.fromPattern("Tracked (.*):");
+    private static final Pattern SPACER_PATTERN = Pattern.compile("[^a-zA-Z\\[\\-\\d].*");
```

Comment:
The traditional way of writing a character group with the literal `-` is to put it last. This way it can't mean a range, so regexp syntax stipulates it must be treated as a literal, even if it is not escaped.

Furthermore, I believe you do not need to escape `[`, since it cannot be mistaken for a character group start. (If you think it improves readability you might do so nevertheless.)

Reply:
We unfortunately do need to escape `[`, it says "Unclosed character class" when you don't. I will rearrange so that `-` is last